src/Command/ProbeWorkerCommand.php

```diff
@@ -121,30 +121,13 @@ protected function execute(InputInterface $input, OutputInterface $output): int
      */
     protected function process(string $data)
     {
-        $timestamp = time();
-
-        if (!trim($data)) {
-            $this->sendResponse(
-                [
-                    'type' => 'exception',
-                    'status' => 400,
-                    'body' => [
-                        'timestamp' => $timestamp,
-                        'contents' => 'Input data not received.',
-                    ],
-                    'debug' => [
-                        'runtime' => time() - $timestamp,
-                        'request' => $data,
-                        'pid' => getmypid(),
-                    ],
-                ]
-            );
+        $data = json_decode($data, true);
+        $timestamp = $data['timestamp'] ?? null;
 
-            return;
+        if ($timestamp === null || !is_int($timestamp)) {
+            throw new \RuntimeException('missing or invalid timestamp');
```

Comment:
Won't this kill the entire worker process instead of returning an error to the master ?

Reply:
Correct. Replaced it.